org.graalvm.shadowed.json/osgi.bnd

```diff
@@ -0,0 +1,9 @@
+Bundle-Description: OSGi-ified version of ${project.name}
+Bundle-Name: ${project.name}
+Bundle-License: Universal Permissive License, Version 1.0
+Bundle-Version: ${project.version}
+Export-Package: \
```

Comment:
Can you please add the Import-Package directive?

I personally would have it here for completeness.

Reply:
So my reasoning for leaving it out where possible:

It further drives home the point that anything we specify in the osgi.bnd file is just _modifying_ whatever was automatically generated from the JAR. So `Import-Package: *` means "just pass it all through untouched". But if you're not really familiar with this (as most developers are not), it looks like it's saying "Import everything under the sun!" (no pun intended!). This is exactly why older versions of this PR had completely unrelated imports - they were copy-pastad from other bundles, and I thought it meant "when inside OSGi, we need to make sure it imports these other packages."

Reply:
I am actually aware that * is just passing through what is inferred by BND Maven on build time.
Your reasoning makes sense, though in this repo all (or at least what I have looked at) bundles specify Import-Package and consistency always is a good thing IMO.